Modules/Sources/WordPressKit/ReaderFeed.swift

```diff
@@ -4,20 +4,56 @@ import Foundation
 /// Encapsulates details of a single feed returned by the Reader feed search API
 /// (read/feed?q=query)
 ///
+/// The API returns different structures depending on the site type:
+/// - WordPress.com sites: Data at root level (URL, title, blog_ID)
+/// - Jetpack sites: Data in meta.data.feed
+/// - External RSS feeds: Data in meta.data.feed, blog_ID is "0"
+///
 public struct ReaderFeed: Decodable {
-    public let url: URL?
-    public let title: String?
-    public let feedDescription: String?
-    public let feedID: String?
-    public let blogID: String?
-    public let blavatarURL: URL?
+    /// Feed ID from meta.data.feed
+    public var feedID: String? {
+        feed?.feedID
+    }
 
-    private enum CodingKeys: String, CodingKey {
-        case url = "URL"
-        case title = "title"
-        case feedID = "feed_ID"
-        case blogID = "blog_ID"
-        case meta = "meta"
+    /// Site/Feed URL with fallback: data.site → data.feed
+    /// Prioritizes site URL over feed URL for canonical representation
+    public var url: URL? {
+        site?.url ?? feed?.url
```

Comment:
I opted to skip reading the fields inlined in the response and use `data.feed` and `data.site` directly as these seem to be more reliable for all three scenarios.